ktor-server/ktor-server-servlet/jvm/src/io/ktor/server/servlet/AsyncServlet.kt

```diff
@@ -42,6 +42,10 @@ public open class AsyncServletApplicationCall(
             putResponseAttribute(it)
         }
     }
+
+    init {
+        servletRequest.putServletAttributesTo(this)
```

Comment:
It's hard to read such API, please consider to swap receiver and argument. `addServletAttributes(servletRequest)`